cdap-docs/developer-manual/source/cloud-runtimes/example/index.rst

```diff
@@ -10,5 +10,125 @@
+For example, to add a secure key named `gcp-key` using curl::
+
+  $ curl localhost:11015/v3/namespaces/default/securekeys/gcp-key -X PUT -d '{
+    "properties": {},
+    "description": "gcp service account key",
+    "data": "{  \"type\": \"service_account\", \"private_key\": \"...\", ... }"
+  }'
+
+The value of the data field is the content of your GCP service account key in string form. 
+
+Create a Profile
+----------------
+
+To create a profile, navigate to the Administration page using the menu at the top right of the CDAP UI.
+From this page, you can view the list of system profiles.
+
+.. figure:: /_images/cloud-runtimes/create-system-profile.png
+  :figwidth: 100%
+  :width: 800px
+  :align: center
+  :class: bordered-image
+
+Click on the button to create a new profile. 
+
+.. figure:: /_images/cloud-runtimes/provisioners.png
+  :figwidth: 100%
+  :width: 800px
+  :align: center
+  :class: bordered-image
+
+Select the Google Cloud Dataproc provisioner.
+
+.. figure:: /_images/cloud-runtimes/provisioner-configure.png
+  :figwidth: 100%
+  :width: 800px
+  :align: center
+  :class: bordered-image
+
+Fill in all configuration fields. For the service account key, specify the name of the secure key that
+was added in the setup part of this walk-through.
+
+Create a Test Pipeline
+----------------------
+
+Click on the green plus button and create a new batch pipeline. Our test pipeline will be a very simple
+one that simply reads a file on GCS and writes to an output directory on GCS. Begin by adding a File source
+and a File sink from the left panel. Connect the source to the sink.
+
+.. figure:: /_images/cloud-runtimes/simple-pipeline-overview.png
+  :figwidth: 100%
+  :width: 800px
+  :align: center
+  :class: bordered-image
+
+Configure the source to read a file from the GCS bucket that you set up earlier.
+
+.. figure:: /_images/cloud-runtimes/simple-pipeline-input.png
+  :figwidth: 100%
+  :width: 800px
+  :align: center
+  :class: bordered-image
+
+Configure the sink to write to a directory that does not yet exist.
+
+.. figure:: /_images/cloud-runtimes/simple-pipeline-output.png
+  :figwidth: 100%
+  :width: 800px
+  :align: center
+  :class: bordered-image
+
+Deploy the pipeline by clicking on the `Deploy` button near the top right of the screen.
+This will bring you to the pipeline overview page. Configure the pipeline to use the
+profile that was created earlier.
+
+.. figure:: /_images/cloud-runtimes/pipeline-profile-manual.png
+  :figwidth: 100%
+  :width: 800px
+  :align: center
+  :class: bordered-image
+
+Note that you can also override any profile setting that was not locked during profile creation.
+
+.. figure:: /_images/cloud-runtimes/pipeline-profile-override.png
+  :figwidth: 100%
+  :width: 800px
+  :align: center
+  :class: bordered-image
+
+Run the pipeline. The pipeline should transition to the `Pending` status while the Cloud Dataproc
```

Comment:
Run the pipeline.  
On the Cloud Dataproc page in the GCP Console, you can monitor the progress of the pipeline. `Pending` means Cloud Dataproc is provisioning the cluster for the run. When completed, the pipeline status transitions to `Starting` > `Running` > `Succeeded`.

Reply:
Is this the Cloud Dataproc Jobs page or the Cluster page that shows this information?

Reply:
I'm going to remove the part about looking at the Dataproc UI and only talk about the CDAP UI. It's too confusing to mention two UIs.